src/theme/ItaliaTheme/_main.scss

```diff
@@ -80,7 +80,18 @@ button.custom-navbar-toggler svg.icon {
     }
   }
 }
-
+.sr-only {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
+  white-space: nowrap;
+  -webkit-clip-path: inset(50%);
+  clip-path: inset(50%);
+  border: 0;
+}
```

Comment:
Questa classe esiste in bootstrap-italia (e in bootstrap 5), ma ha cambiato nome, mi sembra si chiami `visually-hidden` ora